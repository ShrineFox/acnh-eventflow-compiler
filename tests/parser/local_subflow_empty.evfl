flow Root():
    run Local()

local flow Local():
    pass

