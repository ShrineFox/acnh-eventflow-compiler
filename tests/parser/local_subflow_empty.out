RootNode[name=Root, vardefs=[], out_edges=[]]

